com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -366,7 +366,19 @@ internal Vector3 Scale
         /// </summary>
         protected NetworkManager m_CachedNetworkManager;
 
-        private readonly NetworkVariable<NetworkTransformState> m_ReplicatedNetworkState = new NetworkVariable<NetworkTransformState>(new NetworkTransformState());
+        private readonly NetworkVariable<NetworkTransformState> m_ReplicatedNetworkStateServer = new NetworkVariable<NetworkTransformState>(new NetworkTransformState(), NetworkVariableReadPermission.Everyone, NetworkVariableWritePermission.Server);
```

Comment:
We need two replicated state NetworkVariables since these have to be defined during the property declaration (to work properly).